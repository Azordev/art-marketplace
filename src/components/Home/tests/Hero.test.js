import React from 'react';
import {MemoryRouter} from 'react-router';
import renderer from 'react-test-renderer';
import Hero from '../Hero';

it('renders correctly', () => {
  const tree = renderer
      .create(
          <MemoryRouter>
            <Hero />
          </MemoryRouter>,
      )
      .toJSON();
  expect(tree).toMatchSnapshot();
});
