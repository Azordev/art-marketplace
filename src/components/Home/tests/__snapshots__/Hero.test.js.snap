// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="bg-white w-full"
>
  <div
    className="flex items-center w-full"
  >
    <div
      className="flex flex-col lg:flex-row items-center lg:justify-start w-full"
    >
      <div
        className="w-full lg:w-1/2 lg:pl-11 px-5 sm:px-0 py-20 lg:py-0"
      >
        <div
          className="w-full pb-6 space-y-6 sm:max-w-md lg:max-w-lg md:space-y-4 lg:space-y-8 xl:space-y-9 sm:pr-5 lg:pr-0 md:pb-0 mx-auto text-justify lg:text-left"
        >
          <h1
            className="font-bold tracking-add text-lightblack text-2xl sm:text-3xl xl:text-4xl 2xl:text-5xl leading-add text-center lg:text-left"
          >
            <span
              className="block"
            >
              Entre al nuevo mundo
            </span>
            <span
              className="block"
            >
              del Arte Digital
            </span>
          </h1>
          <div
            className="relative flex flex-col sm:flex-row sm:space-x-4 justify-center lg:justify-start"
          >
            <a
              className="shadow-button hover-scale inline-flex items-center justify-center px-11 py-3.5 text-xl font-semibold leading-6 text-white whitespace-nowrap bg-primary rounded-md shadow-sm tracking-add"
              href="/register"
              onClick={[Function]}
            >
              Reg√≠strate ahora
            </a>
          </div>
        </div>
      </div>
      <div
        className="w-full xl:h-full xl:w-1/2 lg:w-3/5 mx-auto"
      >
        <div
          className="w-full h-full"
        >
          <img
            alt="Main Banner"
            className="w-full lg:w-full xl:w-vh"
            src="banner-2.png"
          />
        </div>
      </div>
    </div>
  </div>
</section>
`;
